src/utils/hashStackTrace.ts

```diff
@@ -0,0 +1,32 @@
+import { createHash } from "crypto";
```

Comment:
just a comment for later: `createHash` is one of the functions from crypto that causes a security warning to be issued on a repo. We'll have to disable the warning since it's not used for secure purposes, but for identity purposes.